### PR TITLE
move `moving average` to new pkg

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

move `moving average` to new pkg, only move without modify.

The first step of #3180

### Check List

Tests

- Unit test

### CODE REVIEW

**Review by Yisaer — COMMENTED** [review]
LGTM

File: pkg/movingaverage/moving_average.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package movingaverage
+
+
```
Comment (disksing) [inline pkg/movingaverage/moving_average.go:16]: remove 1 line

File: pkg/movingaverage/time_median.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package movingaverage
+
+import "time"
+
+// TimeMedian is AvgOverTime + MedianFilter
+// Size of MedianFilter should be larger than double size of AvgOverTime to denoisy.
+// Delay is aotSize * mfSize * StoreHeartBeatReportInterval/2
```
Comment (disksing) [inline pkg/movingaverage/time_median.go:20]: StoreHeartBeatReportInterval -> reportInterval?

**Comment by lhy1024** [comment]
@disksing PTAL

File: pkg/movingaverage/time_median.go [hunk 3]
```diff
@@ -17,7 +17,7 @@ import "time"
 
 // TimeMedian is AvgOverTime + MedianFilter
 // Size of MedianFilter should be larger than double size of AvgOverTime to denoisy.
-// Delay is aotSize * mfSize * StoreHeartBeatReportInterval/2
+// Delay is aotSize * mfSize * reportInterval/2
 // and the min filled period is aotSize * StoreHeartBeatReportInterval, which is not related with mfSize
```
Comment (disksing) [inline pkg/movingaverage/time_median.go:21]: this line the same

**Review by nolouch — APPROVED** [review]
lgtm

**Comment by lhy1024** [comment]
@disksing @Yisaer PTAL

**Review by disksing — APPROVED** [review]

**Pull request merged**